configs/common/packages.yaml

```diff
@@ -133,11 +133,11 @@
       variants: +fckit
     shumlib:
       version: [macos_clang_port]
-    faux:
+    fiat:
       version: [develop]
-    # Attention - when updating also check macos site config
-    trans:
+    ectrans:
       version: [develop]
+      variants: ~enable_mkl
```

Comment:
Note. For now, build everthing with lapack and fftw. In a next step, I will add the option to use MKL instead.